git/keys.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 // keySize is the size of generated private keys.
-var keySize = 4096
+var keySize = 2048
```

Comment:
Because:
a) the latency for the user was too long (seconds)
b) some of the tests failed because of the delay
c) the tests took ages